cdxev/amend/operations.py

```diff
@@ -228,3 +229,35 @@ def handle_component(self, component: dict) -> None:
         process_license(
             component, self.list_of_license_names, self.path_to_license_folder
         )
+
+
+class InferCopyright(Operation):
+    """
+    If neither a license nor a copyright is present in a component,
+    this function will create a 'copyright' field in the schema
+    'supplier.name year, all rights reserved'
+    """
+
+    def infer_copyright(self, component: dict) -> None:
+        if "copyright" in component.keys() or "licenses" in component.keys():
```

Comment:
This query checks if the keys are present, would it be more usefull to check if they contain content e.g. 
if component.get("licenses", []) ?

Reply:
I think it's sufficient to check whether any of them exist. Otherwise we could even go a step further and check whether the content of them is correct, so I would start with only checking for the keys. I am not aware of any tool that includes an empty field.